.evergreen/config.yml

```diff
@@ -614,284 +584,296 @@ axes:
-    ruby: "ruby-2.6"
-    mongodb-version: "4.2"
-    topology: standalone
-    os: ubuntu1604
-  display_name: "X.509 Tests"
-  tasks:
-    - name: "x509-tests"
-
--
-  matrix_name: "fle-tests"
-  matrix_spec:
-    auth-and-ssl: "noauth-and-nossl"
-    # Until we stop installing bson from master, don't test on jruby
-    ruby: ["ruby-2.3", "ruby-2.4", "ruby-2.5", "ruby-2.6"]
-    topology: standalone
-    mongodb-version: ["4.2", "4.0"]
-    os: rhel70
-    bson: master
-  display_name: "FLE Tests: ${mongodb-version} ${ruby}"
-  tasks:
-    - name: "fle-tests"
+  - matrix_name: "mongo-4.2"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.7"
+      mongodb-version: ['4.2']
+      topology: "*"
+      os: ubuntu1604
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-4.0"
+    matrix_spec:
+      auth-and-ssl: ["noauth-and-ssl", "auth-and-nossl"]
+      ruby: "ruby-2.6"
+      mongodb-version: ['4.0']
+      topology: "*"
+      os: rhel70
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-3.6"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.5"
+      mongodb-version: ['3.6']
+      topology: "*"
+      os: ubuntu1604
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-3.4"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.4"
+      mongodb-version: ['3.4']
+      topology: "*"
+      os: ubuntu1604
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-3.2"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.3"
+      mongodb-version: ['3.2']
+      topology: "*"
+      os: ubuntu1604
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-3.0"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.7"
+      mongodb-version: ['3.0']
+      topology: "*"
+      os: ubuntu1204
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-2.6"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.6"
+      mongodb-version: ['2.6']
+      topology: "*"
+      os: ubuntu1204
+    display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
+    tasks:
+      - name: "test-mlaunch"
+
+  - matrix_name: "mongo-latest"
+    matrix_spec:
+      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
+      ruby: "ruby-2.7"
```

Comment:
How did you choose which mongo/ruby version pairs to use?

Reply:
I wanted to combine coverage of all server versions and coverage of all ruby versions. The exact pairings are to some extent arbitrary, e.g. if ruby 2.7 is tested with latest then 4.2 could be tested with either 2.7 or 2.6 and 4.0 could be tested with the other one.

Reply:
Makes sense, thanks!